### ISSUE TITLE
HLE Bios

### ISSUE THREAD

**Issue by ADormant** [simias/psx-rs#23]
Will rustation have HLE bios like PCSXR.

**Comment by simias** [comment]
Not in the short term but I'm not opposed to the idea. Does it really improve the experience in PCSXR?

**Comment by ADormant** [comment]
> Does it really improve the experience in PCSXR?

Yes and improves legal situation for users who cannot dump their own bios.
